recipe/meta.yaml

```
@@ -88,6 +88,9 @@ requirements:
     - eigen 3.3.*
     - jasper
     - zlib
+    # specifically compile with svt-av1 to show that
+    # they aren't compatible
+    - svt-av1  1.0
```

```suggestion
    - svt-av1  1.*
```
There is no 1.0, the feedstock releases jumped from 0.9 to 1.1.

---

i'm like 99% sure it is ffmpeg that uses svt-av1.

Opencv fails but it is really failing at importing the ffmpeg libraries.

We internally recompiled our ffmpeg versions for the new svt but i don't have time to fully fix things for conda-forge atm.